docs/latest_release_notes.md

```diff
@@ -1,5 +1,60 @@
 # Latest release notes
 
+## v0.38.0
```

Comment:
I was expecting to be https://github.com/kubevirt/kubevirt/releases/tag/v0.38.1 listed here also?

Reply:
According to the rest of the doc we only populate version info to the first decimal digit.  I'm not sure what we call Z (vX.Y.Z) ... bug fixes or patches?  They get skipped.

Reply:
I think this document should be in sync with x.y.z releases we see on [githubs kubevirt/releases](github.com/kubevirt/kubevirt/releases). However, it makes sense to skip `-rc` etc. postfixed releases for this doc.

Regarding to what to call anything where z > 0 IDK. I _think_ that only bugfixes get merged to release branches, but not completely sure about that.